app/views/setup/welcome.html.slim

```diff
@@ -70,6 +70,24 @@ h1 Initial CaaS Platform Configuration
 
         hr
 
+        .form-group
+          = f.label :flannel_backend, "Flannel backend"
+          = f.select :flannel_backend, options_for_select([["VXLAN", "vxlan"], ["UDP", "udp"]], @flannel_backend), {}, {class: "form-control", 'aria-describedby' => "flannel_backend_help"}
```

Comment:
i just looked at the screenshot, it doesnt seem to be a bootstrap-styled drop down, or is that only your browser?

Reply:
On my computer looks like the select box from [here](http://getbootstrap.com/docs/4.1/components/forms/#form-controls)